test/Billing.Test/Controllers/FreshdeskControllerTests.cs

```diff
@@ -0,0 +1,132 @@
+﻿using System.Text.Json;
+using Bit.Billing.Controllers;
+using Bit.Core.Entities;
+using Bit.Core.Repositories;
+using Bit.Core.Settings;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+using NSubstitute;
+using Xunit;
+
+namespace Bit.Billing.Test.Controllers
+{
+    public class FreshdeskControllerTests
```

Comment:
ASP.net MVC controller classes do some wonky defaulting that makes AutoFixture fall down. As a workaround, the [`ControllerCustomize`](https://github.com/bitwarden/server/blob/EC-307-research-fresh-desk-custom-fields-integration/test/Common/AutoFixture/Attributes/ControllerCustomizeAttribute.cs) customization can be used to automatically generate suts using autofixture.